packages/rancher-logging/generated-changes/overlay/templates/loggings/root/logging.yaml

```
@@ -8,7 +8,7 @@ metadata:
 {{ include "logging-operator.labels" . | indent 4 }}
 spec:
   controlNamespace: {{ .Release.Namespace }}
-  {{- if .Values.global.cattle.windows.enabled }}
+  {{- if and .Values.global.cattle.windows.enabled .Values.nodeAgents.windows.enabled }}
```

```suggestion
  {{- if not (kindIs "invalid" .Values.global.cattle.windows.enabled) }}
  {{- if .Values.global.cattle.windows.enabled }}
```

+ an `{{- end }}` at the very end of the file

---

The reason why you need 2 if statements is because `.Values.global.cattle.windows.enabled` should only be evaluated if the first `if` statement is true, and I believe Helm evaluates both statements in an `{{- if and () () }}` clause, which would cause it to always fail out if `.Values.global.cattle.windows.enabled` is not defined.